GameData/ROEngines/PartConfigs/GEM60_BDB.cfg

```diff
@@ -0,0 +1,110 @@
+PART
+{
+	module = Part
+	name = ROE-GEM60
+	author = Cobaltwolf, Vader111
+
+	RSSROConfig = true
+	ROESetEngineDefaults = SOLID_BOOSTER
+
+	mass = 3.9404
+
+	//	============================================================================
+	//	Update Below
+	//	============================================================================
+
+	MODEL
+	{
+		model = ROEngines/Assets/BDB/Delta/bluedog_Delta_GEM60
+	}
+
+	scale = 1.0
+	rescaleFactor = 1.6
+	node_attach = 0.0, 0.0, -0.46875, 0.0, 0.0, 1.0, 1
+	CoMOffset = 0.0, -0.235, -0.235
+	// stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+	attachRules = 1,1,1,1,0	
+
+	title = GEM 60
+	manufacturer = Orbital ATK
+	description = The 60-inch (1.5-meter) diameter GEM 60 is used on the Delta IV in sets of two or four. When two boosters are used, both are equipped with thrust vector control (TVC). When four are used, two boosters use TVC while the other two use fixed nozzles to reduce weight. Burn time 90 seconds.
+
+	tags = ATK, GEM60, booster, radial, solid, srb,
+
+	engineType = GEM-60
+
+	skinInternalConductionMult = 4.0
+	emissiveConstant = 0.5
+	dragModelType = default
+	maximum_drag = 0.3
+	minimum_drag = 0.2
+	angularDrag = 2
+
+	MODULE
+	{
+		name = ModuleEnginesRF
+		thrustVectorTransformName = thrustTransform
+	}
+
+	MODULE
+	{
+		name = ModuleGimbal
+		gimbalResponseSpeed = 16
+		useGimbalResponseSpeed = true
+		gimbalTransformName = gimbalTransform
+		gimbalRange = 4.0
+	}
+
+	MODULE
+	{
+		name = ModuleB9PartSwitch
+		moduleID = meshSwitchNose
+		switcherDescription = Nose Shape
+		switcherDescriptionPlural = Nose Shapes
+		affectDragCubes = True
+		affectFARVoxels = True
+		SUBTYPE
+		{
+			name = Straight
+			transform = StraightNose
+		}
+		SUBTYPE
+		{
+			name = Angled
+			transform = AngledNose
+		}
+	}
+
+	MODULE
+	{
+		name = ModuleSurfaceFX
+		thrustProviderModuleIndex = 0
+		fxMax = 1
+		maxDistance = 80
+		falloff = 2
+		thrustTransformName = thrustTransform
+	}
+
+	MODULE
+	{
+		name = FXModuleAnimateThrottle
+		animationName = GEM60_Emit_Anim
+		dependOnEngineState = True
+		responseSpeed = 0.001
+	}
+}
+
+@PART[ROE-GEM60]:AFTER[RealismOverhaulEnginesPost] 
+{
+	@MODULE[ModuleEngineConfigs] 
+	{
+		@CONFIG[GEM-60/TVC]
+		{
+			%massMult = 0.9966
+		}
+		@CONFIG[GEM-60/Fixed]
+		{
+			%massMult = 0.9788
+		}
+	}
+}
```

Comment:
If the mass values in the RO engine config are incorrect, please PR the fix there